tests/Unit/OutputFormatTest.php

```diff
@@ -1025,4 +1035,133 @@ public function createCompactReturnsInstanceWithRenderCommentsDisabled(): void
+     */
+    public function createPrettyCalledTwoTimesReturnsDifferentInstances(): void
+    {
+        $firstCallResult = OutputFormat::createPretty();
+        $secondCallResult = OutputFormat::createPretty();
+
+        self::assertNotSame($firstCallResult, $secondCallResult);
+    }
+
+    /**
+     * @test
+     */
+    public function createPrettyReturnsInstanceWithSpaceBeforeRulesSetToNewline(): void
+    {
+        $newInstance = OutputFormat::createPretty();
+
+        self::assertSame("\n", $newInstance->getSpaceBeforeRules());
+    }
+
+    /**
+     * @test
+     */
+    public function createPrettyReturnsInstanceWithSpaceBetweenRulesSetToNewline(): void
+    {
+        $newInstance = OutputFormat::createPretty();
+
+        self::assertSame("\n", $newInstance->getSpaceBetweenRules());
+    }
+
+    /**
+     * @test
+     */
+    public function createPrettyReturnsInstanceWithSpaceAfterRulesSetToNewline(): void
+    {
+        $newInstance = OutputFormat::createPretty();
+
+        self::assertSame("\n", $newInstance->getSpaceAfterRules());
+    }
+
+    /**
+     * @test
+     */
+    public function createPrettyReturnsInstanceWithSpaceBeforeBlocksSetToNewline(): void
+    {
+        $newInstance = OutputFormat::createPretty();
+
+        self::assertSame("\n", $newInstance->getSpaceBeforeBlocks());
+    }
+
+    /**
+     * @test
+     */
+    public function createPrettyReturnsInstanceWithSpaceBetweenBlocksSetToTwoNewlines(): void
+    {
+        $newInstance = OutputFormat::createPretty();
+
+        self::assertSame("\n\n", $newInstance->getSpaceBetweenBlocks());
+    }
+
+    /**
+     * @test
+     */
+    public function createPrettyReturnsInstanceWithSpaceAfterBlocksSetToNewline(): void
+    {
+        $newInstance = OutputFormat::createPretty();
+
+        self::assertSame("\n", $newInstance->getSpaceAfterBlocks());
+    }
+
+    /**
+     * @test
+     */
+    public function createPrettyReturnsInstanceWithSpaceAfterRuleNameSetSpace(): void
+    {
+        $newInstance = OutputFormat::createPretty();
+
+        self::assertSame(' ', $newInstance->getSpaceAfterRuleName());
+    }
+
+    /**
+     * @test
+     */
+    public function createPrettyReturnsInstanceWithSpaceBeforeOpeningBraceSetToSpace(): void
+    {
+        $newInstance = OutputFormat::createPretty();
+
+        self::assertSame(' ', $newInstance->getSpaceBeforeOpeningBrace());
+    }
+
+    /**
+     * @test
+     */
+    public function createPrettyReturnsInstanceWithSpaceAfterSelectorSeparatorSetToSpace(): void
+    {
+        $newInstance = OutputFormat::createPretty();
+
+        self::assertSame(' ', $newInstance->getSpaceAfterSelectorSeparator());
+    }
+
+    /**
+     * @test
+     */
+    public function createPrettyReturnsInstanceWithSpaceAfterListArgumentSeparatorsSetToSpaceForComma(): void
+    {
+        $newInstance = OutputFormat::createPretty();
+
+        self::assertSame([',' => ' '], $newInstance->getSpaceAfterListArgumentSeparators());
```

Comment:
Should this just test that the key `,` has the value {space} - instead of the whole array?

Reply:
Or maybe the test method should be renamed to indicate it's checking the whole array.